CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # openpipelines 0.6.2
 
+## NEW FUNCTIONALITY
+
+* `qc/calculate_qc_metrics`: Add basic statistics to .var and .obs.
```

Comment:
Would it make sense to include which columns the `calculate_qc_metrics` creates, e.g.

* `qc/calculate_qc_metrics`: Add basic statistics to `.var` and `.obs`:
  ```
    var: pct_dropout, num_zero_obs, obs_mean, total_counts
    obs: num_nonzero_vars, pct_{var_qc_metrics}, total_counts_{var_qc_metrics}, pct_of_counts_in_top_{top_n_vars}_vars, total_counts
  ```